### PR TITLE
Option B with more emphasis on embeds.

### PR DESCRIPTION
**Pull request by tomruggs**

### CODE REVIEW

File: cheat-sheet.md [hunk 1]
```diff
@@ -68,29 +65,13 @@ We'll use curl to update our player with this configuration:
...
-    "id": "ad40ec13-dcc3-47e8-8465-4d60b4917bac",
-    "url": "http://players.brightcove.com/2335723943001/ad40ec13-dcc3-47e8-8465-4d60b4917bac_default/index.html"
-}
-```
-
-Grab the `url` and try it out in your browser. At this point, you could use this player on your site by pasting in the `embed_code`.
-
-## Customizing a Player
-You can further customize your player by adding scripts, stylesheets, and plugins. One common reason for customizing a player is to add advertisements. If you're using Doubleclick for advertisments, you would use the [IMA3 plugin](http://docs.brightcove.com/en/video-cloud/players/plugins/ima-plugin.html) to manage ads for your player. We can add the IMA3 plugin by patching the player we've been working on up to this point. First, modify the configuration you've created for the player:
+Creating an embed is similar to configuring a player. In fact you pass in configuration settings that can add to or override settings that you have set up for your player.  Here we will just add.  Open up your text editor and type (or copy) this [JSON](http://www.json.org/) into a file called embed.json:
```
Comment (dmlap) [inline cheat-sheet.md:75]: "Here we will just add." Missing the end of that sentence?

File: cheat-sheet.md [hunk 2]
```diff
@@ -102,89 +83,89 @@ You can further customize your player by adding scripts, stylesheets, and plugin
...
-      "highres": "//players.brightcove.com/2335723943001/service-webinar/oceans-alternate.jpg"
-    }
-  }
+    "embed_code": "<iframe src='//players.brightcove.com/507787973001/07d9f51d-60ae-4b7f-99c2-f81edcc65b95_4ab5e8e3-5924-4496-95ae-1b3d0b4c9ba0/index.html' allowfullscreen webkitallowfullscreen mozallowfullscreen></iframe>",
+    "url": "http://players.brightcove.com/507787973001/07d9f51d-60ae-4b7f-99c2-f81edcc65b95_4ab5e8e3-5924-4496-95ae-1b3d0b4c9ba0/index.html"
 }
 ```
 
-Then apply that configuration to your embed:
+You should see some new fields in the response this time.  Notice there are no longer preview* items.  Instead you will see url and embed_code.  Grab the `url` and try it out in your browser.
```
Comment (dmlap) [inline cheat-sheet.md]: I couldn't track down the other half of that `*`

File: cheat-sheet.md (hunk 1 above)
Comment (tomruggs) [inline cheat-sheet.md:75]: Actually, no.  It's saying that there are only additions.

File: cheat-sheet.md (hunk 2 above)
Comment (tomruggs) [inline cheat-sheet.md]: I can remove it.  I meant it as a wildcard, not a footnote.

**Pull request merged**